src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -39,38 +39,32 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
 
   "The api" should "create a consignment if the correct information is provided" in {
     val expectedResponse: GraphqlMutationData = expectedMutationResponse("data_all")
-    val response: GraphqlMutationData = runTestMutation("mutation_alldata", validAdminToken)
+    val response: GraphqlMutationData = runTestMutation("mutation_alldata", validUserToken())
     response.data.get.addConsignment should equal(expectedResponse.data.get.addConsignment)
 
     checkConsignmentExists(response.data.get.addConsignment.consignmentid.get)
   }
 
   "The api" should "throw an error if the series id field isn't provided" in {
     val expectedResponse: GraphqlMutationData = expectedMutationResponse("data_seriesid_missing")
-    val response: GraphqlMutationData = runTestMutation("mutation_missingseriesid", validAdminToken)
-    response.errors.head.message should equal (expectedResponse.errors.head.message)
-  }
-
-  "The api" should "throw an error if the user id field isn't provided" in {
-    val expectedResponse: GraphqlMutationData = expectedMutationResponse("data_userid_missing")
-    val response: GraphqlMutationData = runTestMutation("mutation_missinguserid", validAdminToken)
+    val response: GraphqlMutationData = runTestMutation("mutation_missingseriesid", validUserToken())
     response.errors.head.message should equal (expectedResponse.errors.head.message)
   }
 
```

Comment:
I think we still need the test to make sure that an admin user can create a consignment as well.

Reply:
Sorry, I've just read the commit comment, ignore me.

Reply:
I considered doing that, but skipped it in the end because I think we're going to have to think about what types of users we need in TDR. I don't think there's any admin user functionality in the MVP plan. There will be backend clients for things like the antivirus check or the series import, but they're not really admin users either - they'll need their own role and restricted permissions.

Reply:
Sorry, I posted before I saw your comment! 🙂